docs/user/gettingStarted/BasicUsage.md

```diff
@@ -40,7 +25,7 @@ Generates data to a specified endpoint.
 * `<profile path>`, a path to the profile JSON file
 * `<output path>`, a file path to where the data should be emitted to. This will be a UTF-8 encoded CSV file or directory, option dependent.
 
-The full list of generate options can be viewed [here](../docs/Options/GenerateOptions.md).
+The full list of generate options can be viewed [here](../commandLineOptions/GenerateOptions.md).
```

Comment:
Can't figure out how to make a comment on generateOptions.md directly but should it have the 

'--allow-untyped-fields' 

flag on it?

It does not appear to be a hidden parameter